spec/features/ingredients/sort_alphabetically_spec.rb

```diff
@@ -0,0 +1,52 @@
+# ### Extension
```

Comment:
This test should not be in a file called `sort_alphabetically_spec` .... the feature that we are testing is for the Ingredients index page, and therefore this test suite should be in there.